project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -105,6 +105,10 @@
             <xpath expr="//field[@name='alarm_ids']" position="attributes">
                 <attribute name="invisible">1</attribute>
             </xpath>
+            <xpath expr="//field[@name='location']" position="before">
+                <field name="recurrent_state" attrs="{'readonly': [('is_task_event','=',True)]}"/>
```

Comment:
this two fields (recurrent_state and recurrent_type) must be always readonly, because they are calculated automatically